Makefile

```diff
@@ -16,13 +16,18 @@
 # The main Makefile
 
 # Hard bootstrap how-to:
-# (only necessary in some cases, for example if you remove some primitive)
+# (only necessary if you remove or rename some primitive)
 #
```

Comment:
This update to the "hard bootstrap how-to" is welcome, but note that these two rounds of bootstrap are needed only because the bootstrap is currently half-broken.  We need to talk about ways to fix it.  My feeling is that it can be fixed easily and that the bootstrap can be further simplified to "half a round".

Reply:
After working on it for a few hours I believe that we cannot un-break it without reverting #324, so we have to choose between:
* one round of bootstrap for every case
* two rounds for removing a primitive and no bootstrap needed when adding one.